safe-api-fetch/src/index.js

```diff
@@ -32,6 +32,13 @@ const clientDataVisitor: SafeAPI.Visitor<ClientDataF> = {
       ...data,
       queryParams: newQueryParams
     };
+  },
+  handleCaptureParam: captureParam => getData => input => {
+    const data = getData(input);
```

Comment:
Can we throw a runtime error that the object needs to be a singleton object? And then add two unit tests that we get runtime errors when called with 1. no fields, and 2. more than two fields.

Reply:
Agree that we need a unit test. The constructor does throw an exception if that isn't the case, so at this point we are guaranteed that the object is a singleton object.